client/signup/config/flows-pure.js

```diff
@@ -18,6 +18,14 @@ export function generateFlows( {
 	getDIFMSiteContentCollectionDestination = noop,
 } = {} ) {
 	const flows = [
+		{
+			name: HOSTING_LP_FLOW,
+			steps: [ 'plans', 'user', 'domains' ],
+			destination: getSignupDestination,
```

Comment:
We don't have to do it in this PR, but the other customisation we might want to do with the `hosting` flow is to skip the site setup that happens after the site is created. That is, if you continue through the site creation process you end up here.
<img width="1396" alt="CleanShot 2023-02-10 at 21 19 15@2x" src="https://user-images.githubusercontent.com/1500769/218041290-ffd7817d-4501-498a-a962-6e855a29f02e.png">

You can change that by changing the destination to something else.

```suggestion
			destination: getDIFMSiteContentCollectionDestination,
```

This will work, although you'll probably want to copy-paste this function and give it a more semantically correct name.